lively.morphic/text/morph.js

```diff
@@ -611,13 +612,14 @@ export class Text extends Morph {
       fontFamily: {
         group: 'text styling',
         type: 'Enum',
-        values: DEFAULT_FONTS.map(f => f.name),
+        values: availableFonts().map(f => f.name),
         defaultValue: 'IBM Plex Sans',
         isStyleProp: true,
         isDefaultTextStyleProp: true,
         after: ['defaultTextStyle'],
         set (fontFamily) {
-          this.setProperty('fontFamily', sanitizeFont(fontFamily));
+          document.fonts.load(`${this.fontStyle} ${this.fontWeight} 12px ${fontFamily}`)
+            .then(() => this.setProperty('fontFamily', sanitizeFont(fontFamily)));
```

Comment:
I am a little worried about placing the asynchronous part inside a setter. This can lead to unexpected behaviour since we usually assume that setting a property is immediately reflected in the state of the object.
Can we maybe defer the rendering instead of deferring the property setting?